template/_package.json

```diff
@@ -72,7 +72,7 @@
     "eslint-plugin-prettier": "^3.0.1",
     "prettier": "^1.16.4"<% } %><% if (ui === 'tailwind') { %>,
     "autoprefixer": "^8.6.4",
-    "tailwindcss": "^0.7.0"<% } else if (ui === 'vuetify') { %>,
+    "tailwindcss": "^1.0.0"<% } else if (ui === 'vuetify') { %>,
```

Comment:
If `postcss` is not used in `tailwindcss` anymore, `autoprefixer` can be removed.

Reply:
Good point!